### PR TITLE
DOCS: Changed shape assignment example to reshape. Elaborated modifying shapes.

### PR DESCRIPTION
**Pull request by petetanru**

### CODE REVIEW

File: doc/source/user/quickstart.rst [hunk 1]
```diff
@@ -628,10 +628,18 @@ An array has a shape given by the number of elements along each axis::
...
+    >>> a.ravel().shape
+    (12,)
+    >>> a.reshape(6,2) # returns the array with a modified shape
+    array([[ 2.,  8.],
+           [ 0.,  6.],
+           [ 4.,  5.],
+           [ 1.,  1.],
+           [ 8.,  9.],
+           [ 3.,  6.]])
+    >>> a.T # returns the array after being transposed
```
Comment (seberg) [inline doc/source/user/quickstart.rst]: unfortunatly `a` does not have the `(6, 2)` since if you remove the .shape assignment, `a.reshape(6, 2)` will return a reshaped array, so if you assign it to a new `a = a.reshape(...)` it would work to replace `a.shape = ...`.

Not sure how its best, but I appreciate that you are taking the trouble of fixing a little bit more then initially.

Comment (petetanru) [inline doc/source/user/quickstart.rst]: Hi, I understand what you're talking about and hence my comment that it returns the array with a new shape rather than manipulate it.. 
I think that's the case for `a.ravel() and a.T` as well. All of the three examples doesn't change `a`

**Comment by seberg** [comment]
Yes, that is fine, but the result print after a.T is from before when a
was actually changed. The result does not fit to the code anymore.

**Comment by petetanru** [comment]
oh right. thanks for the correction. let me try again with a better explanation / accurate code.

**Comment by petetanru** [comment]
Okay I think the code and the explanation should be better now. Trying to figure out how to squash the two commits to one

**Comment by petetanru** [comment]
hey. sorry. this squashing commits from pull request really confuses me so i'll just do a new PR again.
